osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -63,8 +70,11 @@ protected override void Update()
         {
             base.Update();
 
+            float adjustedAlpha = ExponentialAlphaAdjust ? (float)Math.Pow(((Color4)DrawInfo.Colour.Colour).A, 3) : 1;

```

Comment:
Use `AverageColour` to make it work with gradients, or alternatively cube the colour of all four corner colours independently. Lastly, is probably more efficient to use `.Linear` instead of converting to sRGB since that's how it's internally stored. The alpha component isn't touched by the conversion, so behaviour will be the same.